lib/srv/db/server.go

```diff
@@ -609,15 +621,27 @@ func (s *Server) ForceHeartbeat() error {
 // upgrades it to TLS, extracts identity information from it, performs
 // authorization and dispatches to the appropriate database engine.
 func (s *Server) HandleConnection(conn net.Conn) {
-	log := s.log.WithField("addr", conn.RemoteAddr())
+	remoteAddr := conn.RemoteAddr().String()
```

Comment:
Doesn't RemoteAddr include port? Do you need to extract the IP part from address?

Reply:
The `func (s *Server) HandleConnection(` is part of the db agent logic thus in fact the `conn net.Conn` is actually part of connection established from db agent to the the teleport proxy. The `conn.RemoteAddr()`  will return proxy reverse tunnel address instead of IP of the db client.